cmd/bank/main.go

```diff
@@ -1,22 +1,55 @@
 package main
 
 import (
+	"bufio"
 	"context"
+	"database/sql"
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
 	"time"
 
-	"github.com/dogmatiq/dogma"
 	"github.com/dogmatiq/example"
 	"github.com/dogmatiq/example/database"
 	"github.com/dogmatiq/example/messages"
 	"github.com/dogmatiq/example/messages/commands"
 	"github.com/dogmatiq/testkit/engine"
+	"github.com/google/uuid"
 	_ "github.com/mattn/go-sqlite3"
 )
 
 func businessDayFromTime(t time.Time) string {
 	return t.Format(messages.BusinessDateFormat)
 }
 
+func readString(reader *bufio.Reader) string {
+	text, _ := reader.ReadString('\n')
+	return strings.Replace(text, "\n", "", -1)
```

Comment:
I think your second suggestion is more clear.
```suggestion
	return strings.TrimSuffix(text, "\n")
```